environments/openshift/manifests/observatorium-template.yaml

```diff
@@ -1016,13 +1025,15 @@ objects:
     serviceAccountName: prometheus-telemeter
     serviceMonitorSelector:
       matchLabels:
-        app.kubernetes.io/name: prometheus-ams
+        prometheus: ams
```

Comment:
If you merge this, we'll stop picking up the current subscription labels service monitor because it does not have this label

Reply:
Yes, exactly 👍  That's why I created this as a draft for now.